app/dist_src/studio.ini

```diff
@@ -1,9 +1,9 @@
 [general]
 component.name = HTML5StudioV3 ;the application Name
-component.version = 3.12.3.rc2 ;similar to folder name
+component.version = 3.12.3.rc3 ;similar to folder name
```

Comment:
dont you need to make the version by template e.g. {{VERSION}}?

Reply:
After first commit, it won't be the same